HW27/document.js

```diff
@@ -0,0 +1,62 @@
+const myDiv = document.createElement('div');
+myDiv.className = 'buttons';
+myDiv.style.background = 'white';
+
+['Add to friends', 'Write message', 'Job offer', 'Add HW'].map(buttonName => {
```

Comment:
For this task, it is not necessary to create buttons using JS. They could be added directly in HTML.